ci/docker/Dockerfile.build.ubuntu_gpu_tensorrt

```diff
@@ -18,7 +18,7 @@
 #
 # Dockerfile to run MXNet on Ubuntu 16.04 for CPU
 
-FROM nvidia/cuda:10.0-cudnn7-devel
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
```

Comment:
Ubuntu 18 was intentional, I think
@KellenSunderland

Reply:
I'll remove that - and we can move it to a different PR if necessary

Reply:
Not a massive deal either way.  I think both are fairly well supported.